zerver/lib/test_fixtures.py

```diff
@@ -113,8 +140,13 @@ def is_template_database_current(
         settings: str='zproject.test_settings',
         status_dir: Optional[str]=None,
         check_files: Optional[List[str]]=None,
-        check_settings: Optional[List[str]]=None) -> bool:
-    # Using str type for check_files because re.split doesn't accept unicode
+        check_settings: Optional[List[str]]=None) -> int:
+    # This function returns an integer from (3, 2, 1) specifying the type of
+    # state the template db is in and thus the kind of action required.
+    # Return code:- 3 -> Template database is current. No action needed.
```

Comment:
I would return a string, that's one of "current", "needs_rebuild", "needs_migrate", so the API is more readable.

`(needs_rebuild, needs_migrate)`

Reply:
I earlier thought of doing that but later settled for numbers. I guess I will fallback to the strings thing.

Reply:
And rename it to `template_database_status`